src/App/Fossa/VSI/DynLinked/Internal/Resolve.hs

```diff
@@ -0,0 +1,86 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module App.Fossa.VSI.DynLinked.Internal.Resolve (
```

Comment:
Any reason why this is named `Internal`?  This doesn't seem break any sort of module encapsulation, which is normally what haskell modules with `Internal` are made for.

Reply:
I plan to have `App.Fossa.VSI.DynLinked.Analyze`, which uses this and other `App.Fossa.VSI.DynLinked.Internal` modules. 

From there, I wanted to make it clear that `App.Fossa.VSI.DynLinked.Analyze` is the correct entry point, not any of the other stuff, while still exporting a bunch of internal functions that are not really needed except for testing (this effect is much more pronounced in [my ldd parsing PR](https://github.com/fossas/fossa-cli/pull/778)). It's my understanding `Internal` is the correct way to do that.

Reply:
Ok, I just couldn't tell from this PR, but it seems like it's all going to work together.